RULES/cc/cc.go

```diff
@@ -370,7 +370,7 @@ func (lib Library) arRule() core.BuildRule {
 		return core.BuildRule{
 			Name: toolchain.Name() + "-ar",
 			Variables: map[string]string{
-				"command":     fmt.Sprintf("rm -f $out 2> /dev/null; %s rcs --thin $out $in", ninjaEscape(toolchain.Archiver())),
+				"command":     fmt.Sprintf("rm -f $out 2> /dev/null; %s rcs -T $out $in", ninjaEscape(toolchain.Archiver())),
```

Comment:
As far as I could tell from the docs `T` does not have a `-` but I did not try it myself.
should this be `rcsT` instead?

Reply:
Good catch!